controllers/slicegateway/types.go

```diff
@@ -36,6 +36,8 @@ type NetOpPod struct {
 
 type HubClientProvider interface {
 	UpdateNodePortForSliceGwServer(ctx context.Context, sliceGwNodePort int32, sliceGwName string) error
+	UpdateNodeIpInCluster(ctx context.Context, clusterName, nodeIP, namespace string) error
```

Comment:
The cluster name and namespace should not matter to the slice gw. It should be restricted to the hub client pkg. This func should only take ctx and nodeIP as input.